styles.css

```diff
@@ -0,0 +1,139 @@
+.basic, .business, .webshop {
```

Comment:
Do you really need 3 classes?
Can it be just one?

Reply:
done in the last commit